package.json

```diff
@@ -22,7 +22,7 @@
         "css-loader": "^0.28.10",
         "extract-text-webpack-plugin": "^4.0.0-alpha.0",
         "html-webpack-plugin": "^3.0.6",
-        "node-sass": "^4.9.0",
+        "node-sass": "^4.12.0",
```

Comment:
this version was giving me issue (maybe because of my local node version) so i had to upgrade to get the build working